CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,77 @@
+# Contributor Covenant Code of Conduct
```

Comment:
We should have a commone CoC for at least alchemlyb and alchemtest together, although even better would be for all of alchemistry.org . However, that's a different discussion.